lib/features/steps/browser_steps.rb

```diff
@@ -0,0 +1,105 @@
+# @!group Browser steps
+
+When('I navigate to the URL {string}') do |path|
+  $logger.debug "Navigating to: #{path}"
+  MazeRunner.driver.navigate.to path
+end
+
+When('the test should run in this browser') do
+  wait = Selenium::WebDriver::Wait.new(timeout: 10)
+  wait.until {
+    MazeRunner.driver.find_element(id: 'bugsnag-test-should-run') &&
```

Comment:
Do we have some docs about these IDs? I don't think I've seen this before

Reply:
Hmm, I think I need to take another look at this.  I've dropped a number of steps in from the JS repo quite blindly, but a few are quite JS browser specific and I think should move back.  The `bugsnag-test-should-run` element is present in the JS test fixtures, so should make more sense once the step is back in the same repo.

Reply:
Two steps now moved back to the JS repo.  I think there's some further refinement that can be done in time to abstract the JS repo away from Selenium, but I think it's ok for the time being.